### ISSUE TITLE
Messages not being delivered

### ISSUE THREAD

**Issue by Janedoe1951** [signalapp/Signal-iOS#5104]
- [ ] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [ ] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Messages not being delivered to recipients. 

### Steps

type message hit send…. Never shows received and recipient verified no message ever delivered. 

### Device info

**Device**: iDevice X

**iOS version**: X.Y.Z

**Signal version:** Z.Y

### Link to debug log

https://debuglogs.org/dfbbd0086472dc7aeb4b1023403091e4edf24f58cf02a8ecb06e95acf172b18d.zip

**Comment by parcelcat** [comment]
Possibly related to #5072.

**Comment by HeinrichApfelmus** [comment]
I recently experienced the same problem, it started after the update to v5.19.0 . I compose a message, hit "send", and a sound is played, but the message is neither shown nor received by the other party. Messages to self are also affected.

I just noticed that this phenomenon happens only when the phone is connected through my local WLAN network (Other apps can connect to the internet). When the phone uses LTE to connect to the internet, messages work again. This may be a useful temporary workaround.
